lib/page-label/index.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import box from '../box/index';
+
+var pageLabel = angular.module('guPageLabel', ['guBox']);
+
+pageLabel.directive('guPageLabel', function pageLabelDirective(){
+  return {
+    restrict: 'E',
+    transclude: true,
+    replace: true,
+    template: (el, attrs) => `<gu-box variant="tertiary" class="page-label" ng-class="{ open: isOpen }" ng-mouseover="isOpen = true", ng-mouseleave="isOpen = false">
+      <div class="page-label__content" ng-transclude></div>
+    </gu-box>`
+  };
+});
+
+pageLabel.directive('guPageLabelHover', function pageLabelHoverDirective(){
+  return {
+    restrict: 'E',
+    transclude: true,
+    replace: true,
```

Comment:
Also, my usual comment about `replace` being deprecated.

Personally, I think keeping the original tag in the DOM rather than replacing helps with debugging. We should try and re-think how we use `replace` in the future. Perhaps we can support elements and attributes for our directives, allowing multiple components on the same tag.